doc/whatsnew/2/2.15/index.rst

```diff
@@ -30,10 +30,18 @@ Marc Byrne became a maintainer, welcome to the team !
 .. towncrier release notes start
 
 
-What's new in Pylint 2.15.1?
+What's new in Pylint 2.15.2?
 ----------------------------
-Release date: 2022-09-06
+Release date: 2022-09-07
+
+- Fixed a case where custom plugins specified by command line could silently fail.
 
+  Specifically, if a plugin relies on the ``init-hook`` option changing ``sys.path`` before
+  it can be imported, this will now emit a ``bad-plugin-value`` message. Before this
+  change, it would silently fail to register the plugin for use, but would load
+  any configuration, which could have unintended effects.
+
+  Fixes part of #7264. (`#7264 <https://github.com/PyCQA/pylint/issues/7264>`_)
```

Comment:
Can we teach towncrier about whitespaces?

Reply:
What would you expect ?

Reply:
https://github.com/twisted/towncrier/issues/80

Setting `wrap` to `False` should work?